reference/api/tasks.md

```diff
@@ -61,17 +67,17 @@ This value is always `null` for `dumpCreation` tasks.
 
 #### `documentAdditionOrUpdate`
 
-| Name                    | Description                  |
-| :---------------------- | :--------------------------- |
-| **`receivedDocuments`** | Number of documents received |
-| **`indexedDocuments`**  | Number of documents indexed  |
+| Name                    | Description                                                                            |
+| :---------------------- | :------------------------------------------------------------------------------------- |
+| **`receivedDocuments`** | Number of documents received                                                           |
+| **`indexedDocuments`**  | Number of documents indexed. `null` when the task status is `enqueued` or `processing` |
 
 #### `documentDeletion`
 
-| Name                      | Description                     |
-| :------------------------ | :------------------------------ |
-| **`receivedDocumentIds`** | Number of document ids received |
-| **`deletedDocuments`**    | Number of documents deleted     |
+| Name                      | Description                                                                            |
+| :------------------------ | :------------------------------------------------------------------------------------- |
+| **`receivedDocumentIds`** | Number of document ids received                                                        |
```

Comment:
This one is probably changed in another doc PR. It should be named `matchedDocuments` now. Just putting that here as a signal if needed.